### PR TITLE
CommunicationTemplate list component

### LINKED ISSUE #437: Communication template list
This issue aims to track the implementation of the communication template list

### PR DESCRIPTION
**Pull request by jitsedesmet**
This PR aims to close #437.

### CODE REVIEW

**Comment by jitsedesmet** [comment]
After all communication PR's are merged. The will be a cleanup PR.

File: frontend/test/integration/communication/communicationTemplateIndexPage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        const info = render(makeCacheFree(CommunicationTemplateIndexPage));
+
+        const template = getBaseCommunicationTemplate("1");
+
+        await waitFor(() => {
+            mockAxios.mockResponseFor(
+                getQueryUrlFromParams(apiPaths.communicationTemplates, { sort: "name" }),
+                getBaseOkResponse(
+                    getBasePage(apiPaths.communicationTemplates, communicationTemplateCollectionName, [
+                        template,
```
Comment (andpuydt) [inline frontend/test/integration/communication/communicationTemplateIndexPage.test.tsx:34]:
```suggestion
                        template
```

**Review by andpuydt — COMMENTED** [review]
Looks good

**Comment by jitsedesmet** [comment]
You made me mad! Pretties wanted it like that :angry:

**Review by andpuydt — APPROVED** [review]

**Pull request merged**